spec-dtslint/observables/bindCallback-spec.ts

```diff
@@ -0,0 +1,271 @@
+import { bindCallback } from 'rxjs';
+import { a,  b,  c,  d,  e,  f,  g, A, B, C, D, E, F, G } from '../helpers';
+import { SchedulerLike } from '../../src';
+
+describe('callbackFunc', () => {
+  const f0 = (cb: () => any) => {
+    cb();
+  };
+
+  const f1 = (cb: (res1: A) => any) => {
+    cb(a);
+  };
+
+  const f2 = (cb: (res1: A, res2: B) => any) => {
+    cb(a, b);
+  };
+
+  const f3 = (cb: (res1: A, res2: B, res3: C) => any) => {
+    cb(a, b, c);
+  };
+
+  const f4 = (cb: (res1: A, res2: B, res3: C, res4: D) => any) => {
+    cb(a, b, c, d);
+  };
+
+  it('should enforce function parameter', () => {
+    const o = bindCallback() // $ExpectError
+  });
+
+  it('should accept cb 0 param', () => {
+    const o = bindCallback(f0) // $ExpectType () => Observable<void>
+  });
+
+  it('should accept cb 1 param', () => {
+    const o = bindCallback<A>(f1) // $ExpectType () => Observable<A>
```

Comment:
Specifying type parameters explicitly should not be necessary. Usage should be like this:

```ts
const o = bindCallback(f1) // $ExpectType () => Observable<A>
```

Reply:
Hm.. I didn't notice that, Thanks.